decidim-meetings/app/presenters/decidim/meetings/meeting_edition_presenter.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    #
+    # Decorator for meetings in users context
+    #
+    class MeetingEditionPresenter < MeetingPresenter
```

Comment:
I'm not a fan of this name (MeetingEditionPresenter) as it's also used in the 'closing report' feature, but I don't have any alternative proposal 🤷🏽‍♂️